source/_integrations/ecoforest.markdown

```diff
@@ -0,0 +1,65 @@
+---
+title: Ecoforest
+description: Instructions on how to integrate Ecoforest fireplaces with Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2022.9
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@pjanuario'
+ha_domain: ecoforest
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The Ecoforest integration allows monitoring and control of local [Ecoforest](https://ecoforest.com) fireplaces in Home Assistant. These fireplaces are manufactured in Spain.
+
+There is currently support for the following device platforms within Home Assistant:
+
+- [Sensor](#sensor)
+
+{% include integrations/config_flow.md discovery=False %}
```

Comment:
```suggestion
{% include integrations/config_flow.md %}
```
Isn't that the default?

Reply:
No, if I don't include the discovery flag to false it mentions in the text about device discovery wich is not true for this one.